### ISSUE TITLE
User merge

### ISSUE THREAD

**Issue by angaither** [DoSomething/aurora#28]
This could be a beast. But the ability to surface duplicate users and merge them together

**Comment by DFurnes** [comment]
We should revisit this functionality at some point – Northstar is no longer creating duplicate users, but we may still have situations where we want to explicitly merge two accounts (for example, when a user has one account with just an email & another with just a mobile number).

One "gotcha" that we'll have to think about is how we combine references to merged IDs in client apps...

**Comment by angaither** [comment]
the idea of this was to deal with users who had two accounts, not the duplicate user problem

**Comment by DFurnes** [comment]
> the idea of this was to deal with users who had two accounts, not the duplicate user problem

Yeah, I remember talking about that. I'm not sure that's how the feature we had built worked though... it looks like it was only allowing you to merge [when duplicate records were found in searches](https://github.com/DoSomething/aurora/pull/98#issuecomment-126098843).
